UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/BodyPartBlood.cs

```diff
@@ -82,6 +82,14 @@ public partial class BodyPart
 		/// </summary>
 		public float BloodThroughput => bloodThroughput;
 
+		public enum RequiredBlood
```

Comment:
nani

Reply:
also prefer to use the HealthConstants file

Reply:
yeah, was redoing bloodflow stuff, but then changed over to radiation